specification/security/_types/UserProfile.ts

```diff
@@ -28,11 +29,19 @@ export class UserProfileHitMetadata {
   _seq_no: SequenceNumber
 }
 
+export class UserProfileUser {
```

Comment:
How about renaming this type to `UserData`?
```suggestion
export class UserData {
```

Reply:
Ignore the above, you can import `User` from `./types'`:
https://github.com/elastic/elasticsearch-specification/blob/151d0b70a4c514a7986bf8543d13998837b418f4/specification/security/_types/User.ts#L22-L29

Reply:
Unfortunately we can't use `User` here, there is no `enabled` property in this nested object. I can switch the name to `UserData`.

Reply:
Hmmm, using the name here removes our ability to use `UserData` in all of `security`, I think sticking with `UserProfileUser` is best here for that reason?